frontend_layout/css/stylelog.css

```diff
@@ -0,0 +1,283 @@
+body {
+  font-family: 'Lato', Verdana;
+  font-size: 100%;
+}
+
+.img-responsive
+{
+  max-width: 100%;
+  height: auto;
+  float: left;
+}
+
+.wrap
+{
+  width: 100%;  /* ������� ������������ ���� �������� */
+  background: #3538e5;
+  height: 45px;
+}
+
+.wrap2
```

Comment:
I thing you should care about naming.
There are a lot of names having obscure mean.

Reply:
Julia and I will work on it, but I can't fix it now.